src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,73 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  addMovie: (item: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [foundMovie, setFoundMovie] = useState<Movie | null>(null);
+  const [query, setQuery] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [isRequestError, setIsRequestError] = useState(false);
+  const [isErrorMessage, setIsErrorMessage] = useState(false);
+
+  const placeHolderImg
+    = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setIsLoading(true);
+
+    getMovie(query)
+      .then(movie => {
+        if (!('Error' in movie)) {
+          setIsRequestError(false);
+          setIsErrorMessage(false);
+
+          setFoundMovie({
+            title: movie.Title,
+            description: movie.Plot,
+            imgUrl: movie.Poster === 'N/A' ? placeHolderImg : movie.Poster,
+            imdbUrl: `https://www.imdb.com/title/${movie.imdbID}`,
+            imdbId: movie.imdbID,
+          });
+        }
+
+        if (('Error' in movie)) {
+          setIsRequestError(true);
+          setIsErrorMessage(true);
+        }
```

Comment:
this can be rewritten to `if ... else` because of using similar `('Error' in movie)`